Demo/src/main/java/com/braintreepayments/demo/VenmoFragment.java

```diff
@@ -71,32 +87,55 @@ public void launchVenmo(View v) {
 
         boolean shouldVault =
                 Settings.vaultVenmo(activity) && !TextUtils.isEmpty(Settings.getCustomerId(activity));
+        boolean fallbackToWeb = Settings.venmoFallbackToWeb(activity);
```

Comment:
nit: I don't think we need this at all anymore. Same with the settings changes since we now default to this flow.